modules/services/analytics/src/main/java/au/com/shiftyjelly/pocketcasts/analytics/di/TrackerModule.kt

```diff
@@ -35,4 +37,10 @@ object TrackerModule {
     internal fun provideFirebaseAnalytics(@ApplicationContext context: Context): FirebaseAnalyticsWrapper {
         return FirebaseAnalyticsWrapper(FirebaseAnalytics.getInstance(context))
     }
+
+    @Provides
+    @Singleton
```

Comment:
`@Singleton` annotation should be used only with objects that are stateful and should have their state kept while the process is live. Synchronization costs of having a singleton object are greater that instantiation of multiple objects in DI scenarios.